### PR TITLE
Close All Applications

### PR DESCRIPTION
**Pull request by sophiacrennan**
Closed all applications and specified when Spring 2022 applications open for developers

### CODE REVIEW

File: src/content/apply/developer.md [hunk 1]
```diff
@@ -11,9 +11,9 @@ mailLink: "https://gmail.us3.list-manage.com/subscribe?u=3b3ae33f54203ab7a839ae5
 role: Developer
 description: "We're looking for developers to work on research projects with researchers and professors as well as community projects for the Northeastern student body. We have frontend, backend, and full stack developers who are passionate about contributing to the community. As Sandbox continues to grow, we’re building a diverse and skilled team of developers with a variety of experiences, interests, and backgrounds to make amazing software with us."
 isVisible: true
-isOpen: true
-openDate: ""
-closeDate: "May 30th 2021"
+isOpen: false
+openDate: "November 29th 2021"
+closeDate: ""
```
Comment (VKong6019) [inline src/content/apply/developer.md:16]: do we mentioned close date as well? currently, we have it planned to be closed initially on Dec. 6th

File: src/content/apply/apply.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "title": "Join Our Team",
   "subtitle": "Help us transform software development at Northeastern",
-  "applicationStatus": "Fall 2021 Applications are open",
-  "isBannerVisible": true
+  "applicationStatus": "Spring 2022 Applications are open",
```
Comment (VKong6019) [inline src/content/apply/apply.json:4]: wait this is fake news, apps aren't even open yet LOL

Comment (VKong6019) [reply src/content/apply/apply.json:4]: oh wait unless this doesn't show with the `isBannerVisible: false`

**Review by VKong6019 — COMMENTED** [review]
hehe hi

File: src/content/apply/apply.json (hunk 2 above)
Comment (sophiacrennan) [reply src/content/apply/apply.json:4]: ya lol it doesnt show I just updated for when it turns back to true

Comment (sophiacrennan) [reply src/content/apply/apply.json:4]: thinking ahead bro

File: src/content/apply/developer.md (hunk 1 above)
Comment (sophiacrennan) [reply src/content/apply/developer.md:16]: ya I originally put that there but then it felt weird to put the close date when the app isn't even open yet. Ill send you a pic

**Review by VKong6019 — APPROVED** [review]
eggsellent work ops director

File: src/pages/index.js [hunk 3]
```diff
@@ -38,8 +38,7 @@ const IndexPage = ({ data }) => {
         <JoinBannerContent>
           <div>JOIN OUR TEAM</div>
           <span>
-            subscribe to our newsletter to be notified when applications open
-            for spring 2021
+            Developer applications for spring 2021 open November 29th 2021
```
Comment (VKong6019) [inline src/pages/index.js:41]: so sorry i just noticed this but it should say speing 2022

**Review by VKong6019 — CHANGES_REQUESTED** [review]

File: src/pages/about.js [hunk 4]
```diff
@@ -55,8 +55,7 @@ const AboutPage = ({ data }) => {
         <JoinBannerContent>
           <div>JOIN OUR TEAM</div>
           <span>
-            subscribe to our newsletter to be notified when applications open
-            for spring 2021
+            Developer applications for Spring 2021 open November 29th 2021
```
Comment (VKong6019) [inline src/pages/about.js:58]: this too, "Spring 2022"

File: src/pages/index.js (hunk 3 above)
Comment (sophiacrennan) [reply src/pages/index.js:41]: its too late you already approved

**Review by VKong6019 — APPROVED** [review]
yayyyyyyyyyyyyyyyyyyyyyyyy

**Pull request merged**